scripts/rv_expression_association/count_variants.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3  
+
+# This script aims to count the total number of variants (non-ref)
+# from the whole TOB-WGS dataset that are biallelic SNVs
+# and rare (MAF < 5%)
+
+import hail as hl
+from cpg_utils.hail_batch import dataset_path, init_batch
+
+# full TOB-WGS object
+MT = dataset_path('mt/v7.mt')
```

Comment:
Instead of a constant value, could you configure this to take an input path using click, if it's likely that this is relevant for a different MT in future?

Reply:
i tried